src/components/pill-selector-dropdown/PillSelectorDropdown.stories.js

```diff
@@ -254,6 +256,71 @@ export const showAvatars = () => {
     );
 };
 
+export const customPillStyles = () => {
```

Comment:
Should there be a visual regression test for this story?

Reply:
Yes, should be useful. I have a quick follow up PR so I can include it there, since this has been open for a few days now.